### PR TITLE
Starter project with kotlin and kotlintest

### PR DESCRIPTION
**Pull request by christof-vollrath**
Added a demo project with [kotlin](https://kotlinlang.org/) and the test framework [kotlintest](https://github.com/kotlintest/kotlintest)

### CODE REVIEW

File: kotlin/kotlintest/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Open the directory in Idea.
+
+
+## Running Tests
+
+To execute the tests run `gradle test` or run the tests from the IDE you are using (e.g. IntelliJ)
+
+
+## Optimized for Usage of
+- Java 1.1
```
Comment (Egga) [inline kotlin/kotlintest/README.md]: no java here, please 😉

File: kotlin/kotlintest/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Java Truth
+
+Minimal setup with Kotlin and kotlintest to get you started
+
+## Setup
+
+    git clone https://github.com/swkBerlin/kata-bootstraps
+    cd kotlin/kotlintest
+    gradle idea
```
Comment (Egga) [inline kotlin/kotlintest/README.md]: would you mind including a gradle wrapper? i'm on a machine right now without gradle installed...

File: kotlin/kotlintest/src/test/kotlin/ThingTest.kt [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "it should callForAction" {
+        val thing = Thing()
+        val value = thing.callForAction()
+        value shouldBe "Food"
+    }
+
+    "table callForAction" {
+        val thing = Thing()
+
+        val testTable = table(
```
Comment (Egga) [inline kotlin/kotlintest/src/test/kotlin/ThingTest.kt]: nice one 👍

File: kotlin/kotlintest/src/test/kotlin/ThingTest.kt [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            //-----|-----------|------------------------
+            row    ("chocolate", "Dog eats chocolate."),
+            row    ("bone",      "Dog eats fish.")
+        )
+        forAll(testTable) { food, result ->
+            thing.callForAction(food) shouldBe result
+        }
+    }
+
+})
```
Comment (Egga) [inline kotlin/kotlintest/src/test/kotlin/ThingTest.kt]: no newline at end of file 🤷‍♀️

**Comment by christof-vollrath** [comment]
Please merge my pull request, should be useful if somebody likes to do a kata in kotlin.

**Comment by mrksdck** [comment]
Thanks for your work.
Was able to get the tests green, first time used kotlin :D

**Pull request merged**